From 6ff643d615b41a53236255e7d27a43aa35bd24f8 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Wed, 9 Sep 2015 14:09:57 +0200
Subject: [PATCH] ccpp: correct comments mentioning TID

abrt-hook-ccpp uses '%I'.

See commit 25ca1b05529d7ce95a3948f45391e22ab0fd9744 for details.

Related: rhbz#1252590

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/hooks/abrt-hook-ccpp.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/hooks/abrt-hook-ccpp.c b/src/hooks/abrt-hook-ccpp.c
index 5b277af..f5200dc 100644
--- a/src/hooks/abrt-hook-ccpp.c
+++ b/src/hooks/abrt-hook-ccpp.c
@@ -160,13 +160,13 @@ static struct dump_dir *dd;
  * %g - gid
  * %t - UNIX time of dump
  * %e - executable filename
- * %i - crash thread tid
+ * %I - global crash thread tid
  * %% - output one "%"
  */
 /* Hook must be installed with exactly the same sequence of %c specifiers.
  * Last one, %h, may be omitted (we can find it out).
  */
-static const char percent_specifiers[] = "%scpugtei";
+static const char percent_specifiers[] = "%scpugteI";
 static char *core_basename = (char*) "core";
 
 static char* get_executable(pid_t pid, int *fd_p)
@@ -688,7 +688,7 @@ int main(int argc, char** argv)
 
     if (argc < 8)
     {
-        /* percent specifier:         %s   %c              %p  %u  %g  %t   %e          %i */
+        /* percent specifier:         %s   %c              %p  %u  %g  %t   %e          %I */
         /* argv:                  [0] [1]  [2]             [3] [4] [5] [6]  [7]         [8]*/
         error_msg_and_die("Usage: %s SIGNO CORE_SIZE_LIMIT PID UID GID TIME BINARY_NAME [TID]", argv[0]);
     }
-- 
2.4.3

